frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -232,18 +232,17 @@ $tooltip-background-color: #151515;
   z-index: 1;
 }
 
-.co-alert-manager-yaml__explanation {
-  margin-bottom: 0;
-}
-
-.co-alert-manager-yaml__form-entry-wrapper {
-  position: relative;
-}
-
-.co-alert-manager-yaml__form co-file-dropzone__textarea {
-  min-height: 400px;
+.btn-edit-alert-routing {
```

Comment:
I'm confused by this change. It looks unrelated?

Reply:
Think this was a merge issue, it's not used and seems to be an older/duplicate of `co-alert-manager-config__edit-alert-routing-btn`  -I'll remove